doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -266,7 +266,17 @@ Modem libraries
 Libraries for networking
 ------------------------
 
-|no_changes_yet_note|
+* :ref:`lib_nrf_cloud` library:
+
+  * Added:
+
+    * :c:struct:`nrf_cloud_obj` structure and functions for encoding and decoding nRF Cloud data.
+    * :c:func:`nrf_cloud_obj_pgps_request_create` function that creates a P-GPS request for nRF Cloud.
+
+  * Removed:
+
+    * Unused internal codec function ``nrf_cloud_format_single_cell_pos_req_json()``.
+    * :c:func:`nrf_cloud_location_request_msg_json_encode` function and replaced with :c:func:`nrf_cloud_obj_location_request_create`.
```

Comment:
I believe the removed one needs to be without the :c:func: tag and in double backticks, as it will not be there anymore and is not linkable.